app/src/main/java/ch/epfl/sweng/swenggolf/offer/Offer.java

```diff
@@ -20,6 +21,8 @@
     private final String description;
     private final String linkPicture;
     private final String uuid;
+    private double latitude;
```

Comment:
Wouldn't it be better to store a Location object ? Or is it for easier management of the values ?

Reply:
It is easier to store and parcel Strings than a Location object, and we only need the longitude and latitude.

Reply:
After testing the implementation, I found that it is much simpler to leave it like that.